raptor_dbw_can/src/DbwNode.cpp

```diff
@@ -851,6 +854,86 @@ void DbwNode::recvCAN(const can_msgs::msg::Frame::SharedPtr msg) {
     }
 }
 
+// computes values of TireTemp message for a given array of tire temperatures.
+deep_orange_msgs::msg::TireTemp DbwNode::generateTireTemp() {
+    int fl_count = 0, fr_count = 0, rl_count = 0, rr_count = 0;
+    std::vector<float32> fl_temps, fr_temps, rl_temps, rr_temps;
+
+    for (int i = 0; i < 16; i++) {
+        float32 temp = tire_report_msg.fl_tire_temperature[i];
+        if (temp >= 0.0) {
+            tire_temp_report_msg.front_left.mean += temp;
+            fl_count += 1;
+            fl_temps.push_back(temp);
+        }
+
+        temp = tire_report_msg.fr_tire_temperature[i];
+        if (temp >= 0.0) {
+            tire_temp_report_msg.front_right.mean += temp;
+            fr_count += 1;
+            fr_temps.push_back(temp);
+        }
+
+        temp = tire_report_msg.rl_tire_temperature[i];
+        if (temp >= 0.0) {
+            tire_temp_report_msg.rear_left.mean += temp;
+            rl_count += 1;
+            rl_temps.push_back(temp);
+        }
+
+        temp = tire_report_msg.rr_tire_temperature[i];
+        if (temp >= 0.0) {
+            tire_temp_report_msg.rear_right.mean += temp;
+            rr_count += 1;
+            rr_temps.push_back(temp);
+        }
+    }
+
+    sort(fl_temps, fl_temps + fl_count);
+    sort(fr_temps, fr_temps + fr_count);
+    sort(rl_temps, rl_temps + rl_count);
+    sort(rr_temps, rr_temps + rr_count);
+
+    if (fl_count % 2 != 0) {
+        tire_temp_report_msg.front_left.median = static_cast<float>fl_temps[fl_count/2];
```

Comment:
Does this compile? you are missing `()` like it should be `static_cast<float>(fl_temps[fl_count/2]);`